gapis/api/vulkan/state_rebuilder.go

```diff
@@ -607,6 +607,19 @@ func (sb *stateBuilder) createSurface(s SurfaceObjectʳ) {
 			sb.MustAllocWriteData(s.VulkanHandle()).Ptr(),
 			VkResult_VK_SUCCESS,
 		))
+	case SurfaceType_SURFACE_TYPE_GGP:
+		sb.write(sb.cb.VkCreateStreamDescriptorSurfaceGGP(
+			s.Instance(),
+			sb.MustAllocReadData(NewVkStreamDescriptorSurfaceCreateInfoGGP(sb.ta,
+				VkStructureType_VK_STRUCTURE_TYPE_STREAM_DESCRIPTOR_SURFACE_CREATE_INFO_GGP,
+				0, // pNext
+				0,
+				0,
```

Comment:
Comment the two other 0 arguments? (all zeroed arguments are commented in similar calls for other surfaces)